src/config/environment.js

```diff
@@ -0,0 +1,5 @@
+const config = {
+  backendUrl: 'http://localhost:3000/api/v1'
```

Comment:
good catch.   Will fix.  For prod we will need to copy environment-prod.js to environment.js during the build process, before `yarn build`.  I'm looking for a better way to do this.